service/controller/resource/ipam/azure_machinepool_network_range_getter.go

```diff
@@ -0,0 +1,64 @@
+package ipam
+
+import (
+	"context"
+	"net"
+
+	"github.com/giantswarm/microerror"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/giantswarm/azure-operator/v4/pkg/helpers"
+	"github.com/giantswarm/azure-operator/v4/service/controller/key"
+)
+
+var nodePoolIPMask = net.CIDRMask(24, 32)
```

Comment:
any reason to have this variable here instead of just returning the value from the `GetRequiredIPMask` function?

Reply:
Not any strong reason, just for the sake of not creating it every time as it's always the same.

Would it be better to do `return net.CIDRMask(24, 32)` inside the function? More idiomatic maybe?

Reply:
I'm fine either way, I was just curious :smile: